packages/next/client/image.tsx

```diff
@@ -462,7 +509,10 @@ export default function Image({
         {...imgAttributes}
         decoding="async"
         className={className}
-        ref={setRef}
+        ref={(element) => {
+          setRef(element)
+          removePlaceholder(element, placeholder)
```

Comment:
Is there a reason why you're using `ref={removePlaceholder}` instead of `onLoad={removePlaceholder}`?

Reply:
We originally used the `onLoad` implementation but because of https://stackoverflow.com/questions/39777833/image-onload-event-in-isomorphic-universal-react-register-event-after-image-is it wasn't consistently triggering (hence the current `removePlaceholder` handler implementation).

Reply:
I see. Let's add that link in the comment so its clear to the next person reading this code. Here's the short one: https://stackoverflow.com/q/39777833/266535

Reply:
Comment added!